electron/settingsRenderer.js

```diff
@@ -100,6 +105,36 @@ if (settingsData.debug) {
   }, false)
 }
 
+ipcRenderer.on('counterChanged', (e,new_wlc) => {
+console.log(new_wlc)  ;
+  settingsData.winLossObj = new_wlc;
+  settingsData.counterDeckList = counterDecks();
+  settingsData.totalWinLossCounter = settingsData.winLossObj.total;
+});
+
+/*
+ * Format decks in winLossCounter to array for display in rivets.
+ */
+function counterDecks(){
+  let decks = [];
+  let ids = Object.keys(settingsData.winLossObj);
+  for (let x=0;x<ids.length;x++){
```

Comment:
minor: I have a slight preference for `for (let key of ids) { ... }` unless the index is really needed for something. (You can also skip the `Object.keys(...)` call by just doing `for (let key in settingsData.winLossObj)` ; [some docs](https://bitsofco.de/for-in-vs-for-of/))

Reply:
No problem.

Reply:
Done.